### PR TITLE
fix: use jnlp inbound-agent image as source, bump aws-cli version and add yamllint

### PR DESCRIPTION
**Pull request by lemeurherve**
jnlp as source: so we don't need to kubectl exec
bump aws-cli: jnlp Alpine version  doesn't contain 1.18 anymore
add yamllint: global image

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,5 +1,5 @@
-FROM alpine:3.13
-
+FROM jenkins/inbound-agent:4.11-1-alpine-jdk11
```
Comment (timja) [inline Dockerfile:1]:
```suggestion
FROM jenkins/inbound-agent:4.11.2-2-alpine-jdk11
```

**Review by dduportal — APPROVED** [review]
LGTM 🚀 

(failing build is about updatecli: not important)

**Comment by timja** [comment]
> LGTM 🚀
> 
> (failing build is about updatecli: not important)

while that's true, that step is used all over the org so needs fixing to bring a lot of builds back 👅  (unless the updatecli bit is just commented out in pipeline library for now)

**Comment by dduportal** [comment]
> while that's true, that step is used all over the org so needs fixing to bring a lot of builds back 👅 (unless the updatecli bit is just commented out in pipeline library for now)

Correct. The goal of this PR is to enable the main builds first to solve the egg&chicken problem: it will provides updatecli with the new released version (so to solve the updatecli steps on a scond step)

**Comment by dduportal** [comment]
@timja For info: https://github.com/jenkins-infra/docker-helmfile/releases/tag/2.0.0 (it was 1.6.5 but we decided afterwars to bump the release as major)

**Pull request merged**